features/compare.feature

```diff
@@ -19,20 +19,9 @@ Feature: hub compare
     And "open https://github.com/mislav/dotfiles/compare/my%23branch!with.special%2Bchars" should be run
 
   Scenario: No args, no upstream
-    When I run `hub compare`
-    Then the exit status should be 1
-    And the stderr should contain:
-      """
-      Usage: hub compare [-uc] [<USER>] [[<START>...]<END>]
-             hub compare [-uc] [-b <BASE>]
-      """
```

Comment:
This test no longer makes sense.

If I'm on github.com and I fork a repository, then I jump in a terminal and clone my fork, then I push some commits on the default branch, then run `git compare` I should be sent to a comparison of the default branch on my fork, which might suggest I pull request those changes back to the forked repository.